compositor_render/src/renderer.rs

```diff
@@ -184,13 +189,45 @@ impl WgpuCtx {
         let yuv_to_rgba_converter = YUVToRGBAConverter::new(&device, &yuv_bind_group_layout);
         let rgba_to_yuv_converter = RGBAToYUVConverter::new(&device, &rgba_bind_group_layout);
 
+        let shader_parameters_bind_group_layout = Shader::new_parameters_bind_group_layout(&device);
+
+        let global_shader_parameters_buffer = device.create_buffer(&wgpu::BufferDescriptor {
```

Comment:
global is a bit misleading, values here are global to a single render loop or scene, so maybe
- render_ctx_shader_paramaters_buffer
- render_ctx_parameters_buffer (shader in this context is a bit redundant)
- render_loop_paramaters_buffer